weblate/trans/vcs.py

```diff
@@ -630,6 +631,75 @@ def push(self, branch):
 
 
 @register_vcs
+class GithubRepository(GitRepository):
+
+    name = 'GitHub'
+
+    _cmd = 'hub'
+    _hub_config_file = os.path.join(os.path.expanduser('~'),
+                                    '.config/hub')
```

Comment:
I think we can't change the location of this config file:

```
GitHub OAuth authentication
    Hub  will  prompt  for  GitHub  username  & password the first time it needs to access the API and exchange it for an OAuth token, which it saves in
    "~/.config/hub".

    To avoid being prompted, use GITHUB_USER and GITHUB_PASSWORD environment variables.
```